MonoGame.Framework/Media/Video.cs

```diff
@@ -83,7 +83,7 @@ void Dispose(bool disposing)
         {
             if (!_disposed)
             {
-                //PlatformDispose(disposing);
+                PlatformDispose(disposing);
```

Comment:
Not sure why this was disabled.

I put it back...  let the platform specific code handle it internally.